debvader/normalize.py

```diff
@@ -1,32 +1,61 @@
+from abc import ABC, abstractmethod
+
 import numpy as np
 
 
-def linear_normalization_cosmos(x, direction="normalize"):
+class Normalization(ABC):
+    @abstractmethod
+    def forward(self, images):
+        pass
+
+    @abstractmethod
+    def inverse(self, images):
+        pass
```

Comment:
Looks good. 

Few comments on the naming.
- the object itself is not a normalization, but rather a Normalizer. I would call it this way so it is clear
- as I mentioned earlier, both operations should have names that indicate the opposite. It is not clear for me that `inverse` is the opposite of `forward`. It suggest to use either `forward/backward` or the same api as scikit-learn: `transform/inverse_transform`.

Reply:
done!